tests/unit_tests/scheduler/test_scheduler.py

```diff
@@ -342,3 +344,94 @@ async def wait(iens):
 
     assert await sch.execute(min_required_realizations=5) == EVTYPE_ENSEMBLE_STOPPED
     assert killed_iens == [6, 7, 8, 9]
+
+
+@pytest.mark.parametrize(
+    "submit_sleep, iens_stride, realization_runtime, expected_max_running",
+    [(0, 1, 0.1, 10), (0.1, 1, 0.1, 2), (0.1, 2, 0.1, 2)],
+)
+async def test_submit_sleep(
```

Comment:
This funcion maybe represents a strange way to test submit_sleep, by asserting on a correct max_running throughout the simulations which is a consequence of different submit_sleeps found by trial and error more or less.  Perhaps drop this and only test via the function below.

Reply:
resolved.